stunnel/stunnel.ml

```diff
@@ -189,10 +252,13 @@ let attempt_one_connect ?unique_id ?(use_fork_exec_helper = true)
       ["-fd"; if use_fork_exec_helper then config_out_uuid else config_out_fd]
     end in
   let data_out,data_in = Unix.socketpair Unix.PF_UNIX Unix.SOCK_STREAM 0 in
+  (* Dereference just once to ensure we are consistent in t and config_file *)
+  let legacy = !legacy_protocol_and_ciphersuites_allowed in
   let t = 
     { pid = Nopid; fd = data_out; host = host; port = port; 
       connected_time = Unix.gettimeofday (); unique_id = unique_id; 
-      logfile = ""; verified = verify_cert } in
+      logfile = ""; verified = verify_cert;
+	  legacy = legacy } in
```

Comment:
Indentation fault: `legacy` should line up with `logfile`. This is a tab-indented line in the middle of some space-indented lines. (But if this flaw was in the original, let's not fix it in this backport.)